menas/pom.xml

```diff
@@ -65,6 +65,23 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <!-- Spark-avro for AvroSchema conversions -->
+        <dependency>
+            <groupId>org.apache.spark</groupId>
+            <artifactId>spark-avro_${scala.compat.version}</artifactId>
+            <version>${spark.version}</version>
+            <scope>compile</scope>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.slf4j</groupId>
+                    <artifactId>slf4j-log4j12</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>javax.validation</groupId>
+                    <artifactId>validation-api</artifactId>
+                </exclusion>
```

Comment:
Just a question. These artifacts caused binary incompatibility conflicts, right?

Reply:
Honestly, no. I have included the same exclusions that spark-core had (based on some wrong assumptions). After being adviced the `mvn install` part, I was now able to run `mvn dependency:tree` properly to find out that the exclusions do not seem to be needed after all. I am thus removing them.